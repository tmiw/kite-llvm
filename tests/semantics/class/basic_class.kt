class X [
    "initializing X"|print
];

"inside main program"|print;
