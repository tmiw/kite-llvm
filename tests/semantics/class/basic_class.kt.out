initializing X
inside main program
